core/src/main/res/values/strings.xml

```diff
@@ -229,6 +230,7 @@
     <string name="download_type_media">Media file</string>
     <string name="download_type_image">Image</string>
     <string name="download_request_error_dialog_message_prefix">An error occurred when trying to download the file:\u0020</string>
+    <string name="null_value_error_dialog_message_prefix">A null value was found when trying import feed:\u0020</string>
```

Comment:
Why are the two strings so far away from each other? I think it's better to have them next to each other.